source/_docs/installation/synology.markdown

```diff
@@ -2,78 +2,181 @@
 layout: page
 title: "Installation on a Synology NAS"
 description: "Instructions to install Home Assistant on a Synology NAS."
-date: 2016-04-16 11:36
+date: 2018-12-05 10:36
 sidebar: true
 comments: false
 sharing: true
 footer: true
 redirect_from: /getting-started/installation-synology/
 ---
 
-<p class='note warning'>
-Synology only provide Python 3.5.1, which is not compatible with Home Assistant 0.65.0 or later. Until Synology offer an updated version of Python, Home Assistant 0.64 is the most recent version that will be able to be installed. You can manually specify the version of Home Assistant to install, for example to install version 0.64.3 you would do `./python3 -m pip install homeassistant==0.64.3`
+<p class="note warning">
+Synology only provides [Python 3.5.1](https://www.synology.com/en-global/dsm/packages/py3k), which is not compatible with Home Assistant 0.65.0 or later. Until Synology offer an updated version of Python, Home Assistant 0.64 is the most recent version that will be able to be installed. If you want to run newer Home Assistant versions, you will need to make a new Python 3 package, which will be detailed in this guide. Otherwise you can follow the instructions "[Using older Python 3 provided by Synology](#-linkable_title-using-older-python-3-provided-by-synology-)" way below of the document.
```

Comment:
```suggestion
Synology only provides [Python 3.5.1](https://www.synology.com/en-global/dsm/packages/py3k), which is not compatible with Home Assistant 0.65.0 or later. Until Synology offer an updated version of Python, Home Assistant 0.64 is the most recent version that will be able to be installed. If you want to run newer Home Assistant versions, you will need to make a new Python 3 package, which will be detailed in this guide. Otherwise you can follow the instructions "[Using older Python 3 provided by Synology](#using-older-python-3-provided-by-synology)" way below of the document.
```

Reply:
Hmm, yea you are right, when the document is going to be used, the "*linkable_title*" wouldn't exist.

Will correct were used.

Reply:
I think that was the only spot.

Reply:
No, there were other spots, check https://github.com/home-assistant/home-assistant.io/pull/7861/commits/b6cf710e3f08c2bec5a43431f306e334869ca1bb